pilot/pkg/networking/core/v1alpha3/tracing.go

```diff
@@ -369,6 +369,7 @@ func otelConfigGen(_, hostname, cluster string) (*anypb.Any, error) {
 				},
 			},
 		},
+		ServiceName: serviceName,
```

Comment:
This actually makes different OpenTelemetyConfig configurations of different addresses within one proxy config dump all have the same service name , which is it's node cluster name. Is this the correct behavior?

Reply:
this will represent in `localEndpoint.ServiceName`

Reply:
opencensus:
```
2022-11-22T11:36:50.206Z	DEBUG	loggingexporter/logging_exporter.go:52	ResourceSpans #0
Resource SchemaURL: 
Resource labels:
     -> service.name: STRING(sleep.default)
     -> opencensus.starttime: STRING(2022-11-22T11:36:50.203009581Z)
     -> host.name: STRING(sleep-69cfb4968f-drx45)
     -> process.pid: INT(26)
     -> telemetry.sdk.version: STRING(0.5.0-dev)
     -> opencensus.exporterversion: STRING(0.5.0-dev)
     -> telemetry.sdk.language: STRING(cpp)
```

opentelemetry:
```
Resource SchemaURL: 
Resource labels:
     -> service.name: STRING(httpbin.default)
ScopeSpans #0
ScopeSpans SchemaURL: 
InstrumentationScope  
Span #0
    Trace ID       : 5cf4b6b70a6243c504b4dd7cda1c2a60
    Parent ID      : 9bc416e8a6b058ec
    ID             : 08d21cd30af47aa0
    Name           : ingress
    Kind           : SPAN_KIND_SERVER
    Start time     : 2022-11-22 11:35:59.208882 +0000 UTC
    End time       : 2022-11-22 11:35:59.210737 +0000 UTC
    Status code    : STATUS_CODE_UNSET
    Status message : 
Attributes:
     -> node_id: STRING(sidecar~10.244.0.109~httpbin-847f64cc8d-5pnw5.default~default.svc.cluster.local)
     -> zone: STRING()
     -> guid:x-request-id: STRING(125c09f3-8959-9948-b9b7-46ab2ac4b2e0)
     -> http.url: STRING(http://httpbin:8000/ip)
     -> http.method: STRING(GET)
     -> downstream_cluster: STRING(-)
     -> user_agent: STRING(curl/7.86.0-DEV)
     -> http.protocol: STRING(HTTP/1.1)
     -> peer.address: STRING(10.244.0.110)
     -> request_size: STRING(0)
     -> response_size: STRING(28)
     -> component: STRING(proxy)
     -> upstream_cluster: STRING(inbound|80||)
     -> upstream_cluster.name: STRING(inbound|80||)
     -> http.status_code: STRING(200)
     -> response_flags: STRING(-)
     -> istio.mesh_id: STRING(cluster.local)
     -> istio.canonical_service: STRING(httpbin)
     -> istio.namespace: STRING(default)
     -> istio.canonical_revision: STRING(v1)
     -> provider: STRING(otel)
```